tests/baselines/reference/user/puppeteer.log

```diff
@@ -62,6 +50,18 @@ lib/Page.js(185,15): error TS2503: Cannot find namespace 'Protocol'.
 lib/Page.js(394,22): error TS2503: Cannot find namespace 'Protocol'.
 lib/Page.js(407,15): error TS2503: Cannot find namespace 'Protocol'.
 lib/Page.js(731,19): error TS2503: Cannot find namespace 'Protocol'.
+lib/externs.d.ts(2,30): error TS2497: Module '"/puppeteer/puppeteer/lib/Browser"' resolves to a non-module entity and cannot be imported using this construct.
```

Comment:
these just moved around.. looks like we are sorting using case insensitive paths..

Reply:
Platform case, because we're shelling out to `tsc`. I may just resort the errors in the test runner.